.gitignore

```diff
@@ -1,3 +1,4 @@
 /target
 .cargo/config.toml
-/wasm
+wasm/
+dist/
```

Comment:
No `wasm` generated for me and `dist` is in `web`

Reply:
Done